src/formatDate.js

```diff
@@ -50,7 +50,25 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const yearLengthFormat = toFormat.find(el => el[0] === 'Y').length;
+  const dateArray = date.split(fromFormat[3]);
+  const day = dateArray[fromFormat.findIndex(el => el[0] === 'D')];
+  const month = dateArray[fromFormat.findIndex(el => el[0] === 'M')];
+  let year = dateArray[fromFormat.findIndex(el => el[0] === 'Y')];
```

Comment:
repeatable code move to helper function `dateArray[fromFormat.findIndex(el => el[0] ===`